HW13/script.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+
+tput setaf 5; echo "Hello. This is a script for comparing three numbers."
+echo "Enter exit(e) to exit"; tput setaf 7 
+
+while :; do
+tput setaf 4; echo "Enter the number A"; tput setaf 7 
+read A
+if [[ $A == "exit" ]] || [[ $A == "e" ]]; then
+    tput setaf 1; echo "Exit"; tput setaf 7 
+    exit
+fi
+
+tput setaf 4; echo "Enter the number B"; tput setaf 7 
+read B
+if [[ $B == "exit" ]] || [[ $B == "e" ]]; then
+    tput setaf 1; echo "Exit"; tput setaf 7 
+    exit
+fi
+
+tput setaf 4; echo "Enter the number C"; tput setaf 7 
+read C
+if [[ $C == "exit" ]] || [[ $C == "e" ]]; then
+    tput setaf 1; echo "Exit"; tput setaf 7 
+    exit
+fi
+
+declare -i zero=0
```

Comment:
Convert input string number to lowercase, than use `declare`

Reply:
Done!